tests/integration/fixtures/non-updatable-views-filtered.graphql

```diff
@@ -0,0 +1,17 @@
+query {
+  noInsertMutation: __type(name: "InsertNonMutationViewPayload") {
```

Comment:
Yeah that function is perfect for declaratively testing what I am doing, and it will make sure things are not missing as well.